src/com/serotonin/mango/vo/report/ReportChartCreator.java

```diff
@@ -58,6 +58,10 @@ public class ReportChartCreator {
      */
     private static final int IMAGE_WIDTH = 930;
     private static final int IMAGE_HEIGHT = 400;
+	private static final int IMAGE_HEIGHT_FOR_DATA_POINT_NAME_IN_LEGEND_PIXELS = 20;
+    private static final int NUMBER_CHARACTERS_PER_LINE_IN_CHART_LEGEND = 138; //this value refers to IMAGE_WIDTH equal to 930
```

Comment:
We try to avoid comments, the variable name should say what it is, comment is unnecessary. And it does not refer to this parameter at all, the legend parameter does not apply to the entire generated image with a chart;

CHARACTERS_PER_LINE_IN_CHART_LEGEND_NUMBER